graphql/quote.graphql

```diff
@@ -59,6 +59,10 @@ type Quote {
   viewedBySales: Boolean
   viewedByCustomer: Boolean
   salesChannel: String
+  seller: String
+  approvedBySeller: Boolean
```

Comment:
There is no need to have a field only for the seller's approval. A seller quote should use the same status as a regular quote. The only additional information is the seller's name, so you can query this information later.

Reply:
We understand that once the seller accepts the quote, there is no need to make the request to the seller again. `approvedBySeller` starts with `false` when the seller is notified and can become `true` depending on their approval. The first time the marketplace requests this data, it can save it so that it does not need to be requested again if the seller has approved the quote.

Reply:
On second thought, i think you are right. We can just use the status for seller approval control.

Reply:
Done at f6a1117